lesson_28/locators.py

```diff
@@ -0,0 +1,65 @@
+"""Файл с локаторами для всех страниц."""
+
+from selenium.webdriver.common.by import By
+
+# HomePage locators
+HOME_PAGE_LOCATORS = {
+    'guest_log_in_button': (By.XPATH, "//button[@class = 'header-link -guest']"),
+    'menu_dropdown_button': (By.ID, 'userNavDropdown'),
+    'sign_in_button': (By.XPATH, "//button[@class='btn btn-outline-white header_signin']"),
+    'sign_up_button': (By.XPATH, "//button[@class = 'hero-descriptor_btn btn btn-primary']"),
+}
+
+# GaragePage locators
+GARAGE_PAGE_LOCATORS = {
+    'menu_log_out_button': (By.XPATH, "//a[@class='btn btn-link text-danger btn-sidebar sidebar_btn']"),
+    'dropdown_log_out_button': (By.XPATH, "//button[@class = 'dropdown-item btn btn-link user-nav_link']"),
+}
```

Comment:
Acceptable - just note -> As Advanced please read about data classes I think such structures better to organize with data classes